### PR TITLE
Integrate PHPStan

### LINKED ISSUE #214: Null documentElement handling
`documentElement` on the `DOMDocument` class can be either `DocumentElement|null` as noted at https://www.php.net/manual/en/class.domdocument.php

https://github.com/tijsverkoyen/CssToInlineStyles/blob/f4f37098456b0801ee1de84214bbc83078ae3c63/src/CssToInlineStyles.php#L137 doesn't correctly handle when `documentElement` returns `NULL` resulting in:
```console
Argument 1 passed to DOMNode::removeChild() must be an instance of DOMNode, null given 
```

Given valid HTML input, this is noticeable in libxml 2.9.12 due to a bug in libxml https://gitlab.gnome.org/GNOME/libxml2/-/issues/318

### PR DESCRIPTION
**Pull request by bytestream**
Closes #214

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -37,8 +37,10 @@ install:
   - if [[ $setup = 'basic' && "$TRAVIS_PHP_VERSION" != "nightly" ]]; then travis_retry composer install --prefer-dist --no-interaction; fi
   - if [[ $setup = 'lowest' && "$TRAVIS_PHP_VERSION" != "nightly" ]]; then travis_retry composer update --prefer-dist --no-interaction --prefer-lowest --prefer-stable; fi
   - if [[ "$TRAVIS_PHP_VERSION" = "nightly" ]]; then travis_retry composer install --prefer-dist --no-interaction --ignore-platform-reqs; fi
+  - if [[ "$TRAVIS_PHP_VERSION" = "7.4" ]]; then travis_retry composer requrie --dev phpstan/phpstan:1.1.2; fi
 
 script:
+  - if [[ "$TRAVIS_PHP_VERSION" = "7.4" ]]; then composer require vendor/bin/phpstan; fi
```
Comment (stof) [inline .travis.yml:43]: `composer require` looks wrong here

Comment (bytestream) [reply .travis.yml:43]: should be `composer require --dev` ?

Comment (stof) [reply .travis.yml]: should be none of it, as this is expected to run phpstan, not to install it

Comment (bytestream) [reply .travis.yml]: I updated it https://github.com/tijsverkoyen/CssToInlineStyles/pull/215/commits/9f076e68e9aa5663a7aa07f5728ade655760204b

Comment (bytestream) [reply .travis.yml]: Also worth pointing out that travis isn't running anymore... Perhaps should switch to GitHub Actions in another PR?

**Comment by stof** [comment]
Closing in favor of #251 where I introduced phpstan.

I started over from scratch because of the changes in the repo (and in phpstan versions) during those 3 years.

**Pull request closed without merging**